entries/t/trovo.live.json

```
@@ -0,0 +1,13 @@
+{
+  "Trovo": {
+    "domain": "trovo.live",
+    "contact": {
+      "email": "feedback@trovo.live",
+      "facebook": "trovolive",
+      "twitter": "trovolive"
+    },
+    "keywords": [
+      "gaming"
```

Entertainment seems like a better fit.
```suggestion
      "entertainment"
```